### DISCUSSION TITLE
Add custom OrderStatuses (Delivered)

### DISCUSSION THREAD

**Discussion by Web10-Joris** [duncanmcclean/simple-commerce#982] [Feature Requests]
It would be nice to include an 'Delivered' OrderStatus. This would allow an incoming webhook from the delivery service to update the order once it's delivered. If this is too case-specific, providing functionality for adding or creating custom order statuses without modifying source files would be a useful alternative.

**Comment by duncanmcclean** [comment]
Unfortunately, there's no way to add additional cases to enums in PHP (which is what order statuses are).

However, I'll keep this open as a feature request as I may add this in the future.

**Reply by duncanmcclean** [reply]
I've added a new "Delivered" order status which'll ship with v6 (coming in the next few weeks), see #991.

**Reply by Web10-Joris** [reply]
Nice thanks!

**Reply by Web10-Joris** [reply]
I don't have a forked version of SC yet, otherwise I would have created a pull request for it: Don't forget to update/add a case to the `littleDotColour()` in OrderStatusFieldtype.vue & OrderStatusFieldtypeIndex.vue for the delivered status.  :)

**Reply by duncanmcclean** [reply]
Well spotted - added!
